scripts/eurostat/regional_statistics_by_nuts/education_enrollment/README.md

```diff
@@ -70,8 +70,11 @@ The license is [available](https://ec.europa.eu/eurostat/about/policies/copyrigh
 #### Template MCFs

```

Comment:
I'm puzzled about the diffs to scripts/eurostat dir. Do you need to sync to latest?

Reply:
New clean [PR](https://github.com/datacommonsorg/data/pull/316). Closing this one.